docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# CONTRIBUTING
+
+> Before contributing, we highly recommend introducing yourself and your ideas on gitter.[![Gitter](https://badges.gitter.im/codEd-org/simulate.svg)](https://gitter.im/codEd-org/simulate?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+### Setting Up Simulate:
+
+1. Make sure you have flutter installed (https://flutter.dev/docs/get-started/install)
+2. Fork the repository
+3. Clone the forked repository:
+    ```console
+    git clone https://github.com/[YOUR USERNAME]/simulate.git
+    ```
+4. Connect a device or run an emulator (Make sure your connected device has enabled USB Debugging and isn't on 'Charge Only' when connected) 
```

Comment:
enabled USB Debugging --> USB Debugging enabled

Doesn't matter if it's on charge only

Reply:
?

Reply:
It won't matter if it's on charge only. The app would still run.

Reply:
And also, change "Make sure your connected device has enabled USB Debugging" to "Make sure your connected device has USB Debugging enabled"